src/gui/QvisColorTableWindow.C

```diff
@@ -2003,6 +2003,9 @@ QvisColorTableWindow::equalSpacingToggled(bool)
 // 
 //   Justin Privitera, Wed Jun 29 17:50:24 PDT 2022
 //   SetTagChangesMade for copied color tables.
+// 
```

Comment:
In my diffs on GitHub (with whitespace *enabled*), I see only revision *comments* but no actual code changed. Is something missing?

Reply:
I forgot to add modification comments in #17887, so in the description at the top of the PR I mentioned that I was putting them in here instead. I should have made that note clearer. I hope it's ok to put them in this PR instead...